scripts/bnsd/start.sh

```diff
@@ -3,7 +3,7 @@ set -o errexit -o nounset -o pipefail
 command -v shellcheck > /dev/null && shellcheck "$0"
 
 # Choose from https://hub.docker.com/r/iov1/tendermint/tags/
-export BNSD_TM_VERSION=v0.31.5
+export BNSD_TM_VERSION=v1.0.0
```

Comment:
`tendermint` version should be `export BNSD_TM_VERSION=v0.31.11`

Reply:
I saw this "Currently we use the v0.31.5 release" in https://github.com/iov-one/weave README.md

Reply:
Yes, I'm sure about v0.31.11.  I'll update the README.md.